src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -873,6 +873,11 @@ private function registerSessionConfiguration(array $config, ContainerBuilder $c
 
         // session handler (the internal callback registered with PHP session management)
         if (null === $config['handler_id']) {
+            // If the user set a save_path without using a non-default SessionHandler, it will silently be ignored
+            if (null !== $config['save_path']) {
+                throw new LogicException('Session save path is ignored without a handler service');
+            }
```

Comment:
You can/should do this same thing in the `Configuration` class - there is a way to "validate" the options. I would throw an exception for now... until we hear otherwise (you could argue this is a BC break... but why would we ever want people to set an option that's ignored?)

Reply:
Ok, so I've reworked this and still throw an exception, however I added the test/validation to the Configuration as well which is the second check for the issue